access/slack/charts/teleport-access-slack/values.yaml

```
@@ -0,0 +1,54 @@
+##################################################
+# Values that must always be provided by the user.
+##################################################
+
+# The name of Kubernetes service of Teleport.
+# Must be in the same namespace as the plugin.
+teleportService: ""
```

It's unclear what this value should be to me, is this supposed to be the teleport cluster name?
```suggestion
teleportClusterName: ""
```

---

@Joerger No, clusterName differs from service name. It's literally the name of Kubernetes service resource in the same namespace.

So if, for example, you release `teleport-cluster` Helm chart from Teleport repo then the `teleportService` will match the helm release name.